pkg/scheduler/framework/runtime/framework.go

```diff
@@ -766,8 +766,10 @@ func (f *frameworkImpl) RunPostFilterPlugins(ctx context.Context, state *framewo
 		if s.IsSuccess() {
 			return r, s
 		} else if !s.IsUnschedulable() {
-			// Any status other than Success or Unschedulable is Error.
+			// Any status other than Success, Unschedulable or UnschedulableAndUnresolvable is Error.
 			return nil, framework.AsStatus(s.AsError())
+		} else if s.IsUnresolvable() {
+			return r, s
```

Comment:
Merge already handles this; there is no need for special handling unless you are planning to remove `PluginToStatus` completely (and if you do, lets do it as part of https://github.com/kubernetes/kubernetes/pull/114629)

Reply:
Yes, if any postFilter plugin returns `unschedulableAndUnresolvable`, we'll exit immediately.